### ISSUE TITLE
definition() in linked.js does not work

### ISSUE THREAD

**Issue by char101** [harrah/browse#17]
The definition() jquery function in linked.js does not work for non-numeric id.

I suggest changing the definition into

``` javascript
definition: function() {
        var href = this.attr('href');
        if (/#(\d+)$/.test(href)) {
            return $("#" + /#(\d+)$/.exec(href)[1])
        } else if (href != "" && href[0] == '#') {
            href = href.replace(/\./g, '\\.').replace(/;/g, '\\;').replace(/:/g, '\\:')
            return $(href);
        } else {
            return $();
        }
    },
```

**Comment by harrah** [comment]
Thanks, a pull request is welcome.

**Comment by harrah** [comment]
Fixed by #18.
